public/data/akshatmittal61.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "Akshat Mittal",
+  "type": "personal",
+  "bio": "MERN Stack Developer | Competitive Programmer",
+  "avatar": "https://github.com/akshatmittal61.png",
+  "links": [
+    {
+      "name": "Veiw my projects on GitHub",
+      "url": "https://github.com/akshatmittal61",
+      "icon": "github"
+    },
+    {
+      "name": "Mail me to talk",
+      "url": "mailto:akshatmittal2506@gmail.com",
+      "icon": "email"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/akshatmittal61",
+      "icon": "twitter"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/akshatmittal61/",
+      "icon": "linkedin"
+    },
+    {
+      "name": "See my posts on InG",
```

```suggestion
      "name": "See my posts on Instagram",
```
```suggestion
      "name": "See my posts on IG",
```
"InG" looks a bit weird 😅